src/V3AstNodeExpr.h

```diff
@@ -622,7 +626,7 @@ class AstCExprUser final : public AstNodeExpr {
     bool isGateOptimizable() const override { return false; }
     bool isOutputter() override { return true; }
     bool isPredictOptimizable() const override { return false; }
```

Comment:
At this point you might as well make `isGateOptimizable` and `isPredictOptimizable` the same as in `AstCExpr` ?